FABulous/fabric_generator/file_parser.py

```diff
@@ -829,6 +829,9 @@ def parseBelFile(
             else:
                 internal.append((portName, direction))
 
+            if portName[-1].isdigit():
+                individually_declared = True
```

Comment:
I think we should move away from "individually declare" since this is just not natural to write and also is valid to have a digit as the character of the name. This will do for now.

Reply:
Yes, this is not really a good approach in general, which can also cause problems.
This was just a quick fix to get the fabric running again.

Reply:
Let's mark it with a TODO/FIXME.

Reply:
Yes, I'll add a TODO there.